### PR TITLE
Fix part of #2833: Missing named arguments to BindableAdapterTest.

### PR DESCRIPTION
**Pull request by jkugsiya**
## Explanation

Fixes part of #2833: Add missing named argument in BindableAdapterTest

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt [hunk 1]
```diff
@@ -161,7 +161,7 @@ class BindableAdapterTest {
         assertThat(recyclerView.childCount).isEqualTo(1)
       }
       // Perform onView() verification off the the main thread to avoid deadlocking.
-      onView(atPosition(R.id.test_recycler_view, 0))
+      onView(atPosition(R.id.test_recycler_view, position = 0))
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt]: add named argument for recyclerview id too.  
```suggestion
      onView(
        atPosition(
          recyclerViewId = R.id.test_recycler_view,
          position = 0
        )
      )
```

Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt]: ditto for all

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt [hunk 2]
```diff
@@ -440,12 +445,17 @@ class BindableAdapterTest {
...
+          R.id.test_recycler_view,
+          position = 0
+        )
+      ).check(matches(withText("new value")))
     }
   }
 
   private fun setUpTestApplicationComponent() {
-    ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
+    ApplicationProvider.getApplicationContext<TestApplication>().inject(bindableAdapterTest = this)
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt]: let's not add named argument for this.

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt [hunk 3]
```diff
@@ -560,7 +570,7 @@ class BindableAdapterTest {
   private fun inflateTextViewForStringWithoutDataBinding(viewGroup: ViewGroup): TextView {
     val inflater = LayoutInflater.from(ApplicationProvider.getApplicationContext())
     return inflater.inflate(
-      R.layout.test_text_view_for_string_no_data_binding, viewGroup, /* attachToRoot= */ false
+      R.layout.test_text_view_for_string_no_data_binding, viewGroup, /* attachToRoot= */false
```
Comment (anandwana001) [inline app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt]:
```suggestion
      R.layout.test_text_view_for_string_no_data_binding, viewGroup, /* attachToRoot= */ false
```

**Review by anandwana001 — CHANGES_REQUESTED** [review]

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt (hunk 1 above)
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt]: added named arguments for recyclerViewId : https://github.com/oppia/oppia-android/pull/3153/commits/431cf43838b96dca47cfe23de07f460bd53d4ed3

File: app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt (hunk 2 above)
Comment (jkugsiya) [reply app/src/sharedTest/java/org/oppia/android/app/recyclerview/BindableAdapterTest.kt]: Removed named argument for bindableAdapterTest: https://github.com/oppia/oppia-android/pull/3153/commits/431cf43838b96dca47cfe23de07f460bd53d4ed3

**Comment by anandwana001** [comment]
@jkugsiya follow - https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @jkugsiya

**Review by BenHenning — APPROVED** [review]
Thanks @jkugsiya! I have nothing to add--LGTM.

**Pull request merged**